internal/terminal/log.go

```diff
@@ -75,6 +76,16 @@ func NewErrorLog(err error) Log {
 	return newLog(LogLevelError, errorMessage{err})
 }
 
+// NewWarningLog creates a new warn log
+func NewWarningLog(format string, args ...interface{}) Log {
+	message := format
```

Comment:
might be worth extracting this to an unexported helper func within this file (share it b/w textlog and warninglog)